packages/api/src/core/index.ts

```diff
@@ -26,7 +26,7 @@ export { getJSONSchemaDefaults, parseResponse, prepareAuth, prepareParams, prepa
 export default class APICore {
   spec: Oas;
 
-  private auth: (number | string)[][] = [];
+  private auth: (number | string)[] = [];
```

Comment:
Did i typescript correctly?

Reply:
Yep. I updated `prepareAuth.test.ts` in https://github.com/readmeio/api/pull/431/commits/091049816549923dc1069df2ea6d679fa4d075f1 to account for this.